people/views.py

```diff
@@ -19,18 +20,21 @@ def person(request, login):
         except PersonalData.DoesNotExist:
             personal_data = PersonalData(person=person)
             personal_data.save()
-        people = Person.objects.order_by('login')
         return render(request,
                       'people/person.html',
-                      {'person': person,
+                      {'found': True,
+                       'person': person,
                        'personal_data': personal_data,
                        'people': people,
                        'search_form': search_form})
     except Person.DoesNotExist:
         try:
             path = '/{login}/'.format(login=login)
-            resolve(path)   # If this fails, it will raise Resolver404.  Otherwise we just redirect to that URL.
+            resolve(path)  # If this fails, it will raise Resolver404.  Otherwise we just redirect to that URL.
             return HttpResponseRedirect(path)
         except Resolver404:
             pass
-        raise Http404("Person with id '{}' does not exist.".format(login))
+        return render(request, 'people/person.html', {
+            'person': 'User not found',
```

Comment:
```suggestion
            'person': None,
```
I suggest returning None or an object instead of sometimes returning an object and when not found, returning a string